api/resources/translations/messages-ne.properties

```diff
@@ -989,365 +1058,291 @@ related_entities.clinic.parent.parent.name = जिल्ला अस्पत
+tasks.group.completed = सबै घरपरिवारका कार्यहरू पूरा भए।
+tasks.group.title = अन्य घरपरिवारका कार्यहरू
+testing.description = हामीले लुकाएर राखेको परिक्षण पृष्ठ तपाईंले भेट्नु भएको छ। आफ्नो इन्स्टलेसन बिगार्न चाहन्नु हुन्न भने फिर्ता जानुहोस्। 
+testing.title = परिक्षण पृष्ठ
+title = शिर्षक
 to\ recipient = पाउने {{recipient}}
 today = आज
-tomorrow =
+tomorrow = भोलि
 training_cards.confirm.exit = यो तालिम सकिएको छैन । यदि तपाईंले अहिले छोड्नुभयो भने, तपाईंले आफ्नो प्रगति गुमाउनु हुनेछ र यसलाई पूरा गर्न पछि फेरि सम्झाइनेछ।
 training_cards.confirm.button.no = रद्द गर्नुहोस्
 training_cards.confirm.button.yes = छोड्नुहोस्
 training_cards.confirm.title = तालिम छोड्ने हो?
-translation.add =
-translation.key = 
-unique.id = 
-upgrade = 
-upgrade.description = 
-upload = 
-upload.favicon = 
-upload.header.logo = 
-upload.icon = अपलोड चिह्न
-upload.json.forms.help = फाइल फारम परिभाषा एक जेएसओएन इनकोडिंग सूची समावेश गर्नुपर्छ । कुनै पनि अवस्थित प्रकारका अधिलेखित गरिनेछ ।
-upload.json.forms.title = 
-upload.sms.forms = 
-upload.xform.context = 
-upload.xform.help = 
-upload.xform.meta.help = 
-upload.xform.title = 
-upload.xform.xml = 
-upload.xform.xml.help = 
-url = 
-user.fullname.help = 
-user.password.current = 
-user.phone.help = 
-user.place.help = 
-user.username.help = 
-username.invalid = 
-username.taken = 
-usertype._admin = 
+training_cards.error.loading = तालिम लोड गर्दा त्रुटी भयो। तपाइँको सुपरभाइजरलाई सम्पर्क गर्नुहोस्। 
+training_cards.error.save = तालिम सेभ गर्न त्रुटी।
+training_cards.form.saved = तालिम सम्पन्न भयो। 
+training_cards.modal.title = महत्वपुर्ण परिवर्तनहरु
+translation.add = नयाँ अनुवाद कुञ्जी थप्नुहोस्
+translation.key = अनुवाद कुञ्जी
+unique.id = आईडी
+unknown.contact = अपरिचिती सम्पर्क 
+upgrade = अपग्रेड
+upgrade.description = तपाईँको एपलाई कुनै रिलीज, बीटा, वा ब्रान्चमा अपग्रेड गर्न, पहिलो चरणको रूपमा स्टेज गर्न सिफारिस गरिन्छ। यसले प्रयोगकर्ताहरूलाई अवरोध नगरी पृष्ठभूमिमा इन्स्टलेसन तयारी गर्छ। स्टेजिंग पुरा भएपछि, अपग्रेडको लागि 'Install' क्लिक गर्नुहोस्। यो पछि, यसलाई पहिलाकै अवस्थामा  सकिँदैन, त्यसैले कृपया तपाईँको डेटा ब्याकअप गरिएको छ र तपाईँका प्रयोगकर्ताहरूलाई डाउनटाइमको जानकारी दिइएको छ भन्ने निश्चित गर्नुहोस्।
+upload = अपलोड
+upload.favicon = आइकन
+upload.header.logo = शिर्षक (header) को लोगो
+upload.icon = आइकन अपलोड गर्ने
+upload.json.forms.help = फाइल फारम परिभाषा एक JSON इनकोडिंग सुची समावेश गर्नुपर्छ । पहिल्यै भएका कुनै कुरा अध्यवाधिक गरिने छ।
+upload.json.forms.title = JSON फर्महरू अपलोड गर्नुहोस् 
+upload.sms.forms = SMS फर्महरू अपलोड गर्नुहोस् 
+upload.xform.context = XForm सन्दर्भ अपलोड गर्नुहोस् 
+upload.xform.help = XForm अपलोड सहायता
+upload.xform.meta.help = XForm मेटाडाटा अपलोड सहायता
+upload.xform.title = XForm अपलोड गर्नुहोस्
+upload.xform.xml = XForm XML अपलोड गर्नुहोस्
+upload.xform.xml.help = XForm XML अपलोड सहायता
+url = URL
+user.fullname.help = प्रयोगकर्ताको पूरा नाम
```

Comment:
```suggestion
user.fullname.help = हाल एपमा कतै देखाइएको छैन।
```

Reply:
There is wrong translation in the english for this key, and we shouldn't translate exactly that based on how it's used. I'll go ahead and update the english and other translation meaning as well.